packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -1096,8 +1095,8 @@ export class MgtPersonCard extends MgtTemplatedTaskComponent implements IHistory
   };
 
   private get hasPhone(): boolean {
-    const user = this.personDetails as User;
-    const person = this.personDetails as microsoftgraph.Person;
+    const user = this.internalPersonDetails as User;
+    const person = this.internalPersonDetails as microsoftgraph.Person;
```

Comment:
```suggestion
    const person = this.internalPersonDetails as Person;
```
Where is `microsoftgraph` coming from?

Reply:
Oh, I figured it from the MS types package. It's a namespace name. Still, we can avoid using it like this cos Person is imported already.